templates/static_analysis/static_analysis.html

```diff
@@ -891,6 +892,17 @@ <h3 class="box-title"><i class="fa fa-bug"></i> DEX Malware Analysis</h3>
                               {% endfor %}
                           {% endif %}
                         </td>
+                       
+                        <td>
+                          {% if apkid|key:"anti_debug" == "" %}
```

Comment:
`if apkid|key:"anti_debug"|length == 0` is better ?

Reply:
need to dicsuss cause we need in this case to rewrite all the file, I have done a simple copy/paste

Reply:
I think it's fine for now to use `{% if apkid|key:"anti_debug" == "" %}`It won't break any functionality.
This file has more occurance of the same coding style. We can take care of that in a Code QA later, or do a global replace if it doesn't break anything and if it's not too much work to change and test.